_topics/en/silent-payments.md

```diff
@@ -66,12 +66,29 @@ see_also:
 ## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
 ## Should be less than 500 characters
 excerpt: >
-  **Silent payments** are a type of payment that can be made to a
-  unique onchain address for every payment even though the receiver
-  provided the spender with a reusable (offchain) address.  This helps
-  improve privacy.
+  **Silent payments** unlocks the ability to receive private payments from
+  anyone on a single static address without requiring any interaction or
+  extra on-chain overhead. This helps improve privacy.
```

Comment:
Updated to be a bit more accurate, the original excerpt mentions the provided address is off chain.

This is not always the case, see the signet address provided in BIP352 [here](https://github.com/bitcoin/bitcoin/issues/28536#issuecomment-1735484152).

Reply:
Thank you for the detailed and thoughtful response! I agree, will revert the change here